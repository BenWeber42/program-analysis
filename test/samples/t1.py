
def f(x, y):
    return (2+x, 12+x+y)

def f_inv(u, v):
    a = unknown_choice(u,v,0,-u,-v) + unknown_choice(u,v,0,-u,-v) + unknown_int()
    b = unknown_choice(u,v,0,-u,-v) + unknown_choice(u,v,0,-u,-v) + unknown_int()
    return a, b
