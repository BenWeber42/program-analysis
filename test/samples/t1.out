2 12
3 14
4 16
5 18
6 20
7 22
